docs/docs/api/09-Remote-Device-Controller.md

```diff
@@ -0,0 +1,114 @@
+
+2. Ensure that you have the required import `UiController`.
+
+        import { UiController } from 'askui';
+
+3. Also add the global variable if it is missing:
+
+        let uiController: UiController;
+
+4. Modify the creation of `uiController` in file `test/helper/jest.setup.ts`:
+
+        uiController = new UiController({
+          /**
+            * Select the display you want to run your tests on, display 0 is your main display;
+            * ignore if you have only one display
+            */
+          display: <YOUR_DISPLAY_ID>,
+          binaryVersion: "tdk-23.10.1-preview-release-1"
+        });
+
+   > Note: The ui controller creation starts typically with `uiController = await UiController({ ...` but the instance name might be different.
+
+5. Replace `<YOUR_DISPLAY_ID>` with the display (starting with 0 for the first display) that you want to use:
+
+        display: <YOUR_DISPLAY_ID>,
+
+6. Ensure that the controller gets started before the UI controller client tries (`aui`) to connect. Therefore the last two lines of the function should do:
+
+       await uiController.start();
+       await aui.connect();
+
+    > Note: The ui controller client creation starts typically with `aui = await UiControlClient.build({ ...` but the instance name might be different.
+
+7. Ensure that in function `afterAll` the controller gets stopped after the ui controller client closes the connection:
+
+        aui.close();
+        await uiController.stop(true);
+
+8. You should be good to go now to run your workflows as described in documentation.
+
+> Note: New features are only supported on Windows 10/11.
+
+## Features
+
+### Type Like a Human
+Together wit the asynchronous execution of actions we also introduce natural typing. The 
+first iteration of this feature uses the typical typing speed of a human.
+
+### Experimental Unicode Support
+Typing Unicode text is supported but is not yet production ready.
+
+### Remote Device Controller Singleton Process
+The native process (AskuiRemoteDeviceController) is a singleton  process. As soon as a new 
+instance gets started the previous one gets killed. Later this will be managed by an 
+operating system service to avoid interruptions of running workflows.
+
+### Android on Windows
+AskUI automation works on an Android device that is controlled from a Windows machine. It allows you to run a workflow on a Android device, including recording the session. Instructions how to setup the Android workflow can be found in our [documentation](../general/04-Executing%20Automations/mobile-automation.md#android-automation]).
+
+> Please also read the knows issues section since this is just a preview feature.
+
+## Known Issues
+
+### Mouse Movement
+- You may encounter issues when mouse pointer acceleration is enabled.
+
+### Log Files
+- Log files disk space usage increased. They are stored in `<YOUR_USER_FOLDER>/.askui/` and can be deleted if that causes any issues.
```

Comment:
Is it important to the user that with one release the "Log files disk space usage [was] increased"?

Reply:
Actually no. Good catch. But I will leave the **Log Files** disk usage in there.